data/json/regional_map_settings.json

```diff
@@ -24,6 +28,10 @@
           "boosted_other": {                "//": "...and the first non-default picked from this list...",
               "t_shrub_blueberry": 40.0,
               "t_shrub_strawberry": 30.0,
+              "t_shrub_rose": 5.0,
+              "t_shrub_hydrangea": 3.5,
+              "f_flower_tulip": 3.5,
+              "f_flower_spurge": 5.0,
```

Comment:
![image](https://user-images.githubusercontent.com/16213433/45268505-1d993f80-b486-11e8-8a4a-40d0cf38d8e8.png)

Reply:
This _should_ be good once the other PR is merged first as I lowered these numbers in the other PR.